spec/models/kalibro_range_spec.rb

```diff
@@ -57,15 +74,37 @@
           expect(subject.as_json["end"]).to eq("INF")
         end
       end
+    end
+  end
 
-      context 'with negative infinity end' do
-        before do
-          subject.end = -Float::INFINITY
-        end
+  describe 'setters' do
```

Comment:
What about the third case? The one which the parameter is neither `INF` nor `-INF`. Should it be covered?